test/groovy/DockerConfigTest.groovy

```diff
@@ -0,0 +1,92 @@
+
+import org.junit.Test
+import io.jenkins.infra.DockerConfig
+import io.jenkins.infra.InfraConfig
+
+import static org.junit.Assert.assertEquals
+import static org.junit.Assert.assertFalse
+import static org.junit.Assert.assertTrue
+
+class DockerConfigTest {
+
+    static String testImageName = "bitcoinMinerImage"
```

Comment:
Maybe better to rename to prevent scanner tools from grumbling at some point

Reply:
done